cpp/config/IceTargets.cmake

```diff
@@ -52,6 +52,42 @@ find_path(Ice_SLICE_DIR
   NO_DEFAULT_PATH
   REQUIRED)
 
+# Find all executables in the native/bin directory and create imported targets
+if(WIN32)
```

Comment:
For my understanding: WIN32 means Windows for cmake, not the Win32 platform (aka x86)?